controller/scheduler/swarm.py

```diff
@@ -48,14 +63,20 @@ def stop(self, name):
         """
         Stop a container
         """
-        self.docker_cli.stop(name)
+        try:
+            self.docker_cli.stop(name)
+        except:
+            pass
```

Comment:
It doesn't matter if we dont raise runtime error as we any ways check the state of the container and raise runtime error. But I'll change the code any ways as this code might be avoiding some corner cases